app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_of_file = command.split()
+    if name_of_file[1] == name_of_file[2] or name_of_file[1][-4:] != ".txt":
+        return
+
+    from shutil import copyfile
```

Comment:
I see the first time import in the middle of the file, but maybe it's ok, I am not sure)